server/db/models/orders.js

```diff
@@ -0,0 +1,24 @@
+// define Orders model here
+const Sequelize = require('sequelize');
+const { db } = require('./../database.js');
```

Comment:
the bracket notation is not needed here, I believe:

**const db  = require('./../database.js');**